documentation/asciidoc/topics/proc_creating_external_services.adoc

```diff
@@ -1,29 +1,53 @@
 [id='exposing_external_services-{context}']
-= Creating External Services
-To connect to {brandname} clusters from clients running outside {k8s}, create
-external services with {ispn_operator}.
+= Exposing {brandname} to External Clients
+Expose {brandname} clusters to clients running outside {k8s} with external
+services.
 
 .Procedure
 
-. Modify your custom resource definition to configure an external service and then apply the changes.
+* Modify your custom resource definition to configure an external service and then apply the changes.
```

Comment:
User can't modify custom resource definition. What modify your Infinispan resource?

Reply:
@Crumby thanks Pavel. I've made a bunch of changes to clarify the wording here. it probably expands the scope of the PR a bit but we might as well apply these changes across the board.